openesdh-repo/src/main/java/dk/openesdh/repo/webscripts/utils/WebScriptUtils.java

```diff
@@ -104,19 +96,16 @@ public static JSONObject readJson(WebScriptRequest req) {
     public static Object readJson(Class<? extends Object> clazz, WebScriptRequest req) throws IOException {
         checkContentTypeJson(req);
         MappingJackson2HttpMessageConverter converter = new MappingJackson2HttpMessageConverter();
-        Version version = new Version(1, 0, 0, "SNAPSHOT", "dk.openesdh", "multi-language-value-deserializer");
-        SimpleModule module = new SimpleModule("MultiLanguageValueDeserializer", version);
-        module = module.addDeserializer(MultiLanguageValue.class, new MultiLanguageValueDeserializer());
-        converter.getObjectMapper().registerModule(module);
+        ObjectMapper objectMapper = converter.getObjectMapper();
+        objectMapper.registerModule(MultiLanguageValueDeserializer.getDeserializerModule());
+        objectMapper.registerModule(PersonInfoDeserializer.getDeserializerModule());
+        objectMapper.registerModule(CaseFolderItemDeserializer.getDeserializerModule());
```

Comment:
maybe this "converter" can be created as static final and not created on every call?